qt/aqt/overview.py

```diff
@@ -285,7 +287,17 @@ def _renderBottom(self) -> None:
             # links.append(["F", "cram", _("Filter/Cram")])
         if self.mw.col.sched.have_buried():
             links.append(["U", "unbury", tr.studying_unbury()])
-        links.append(["", "description", tr.scheduling_description()])
+
+        QueryOp(
+            parent=self.mw,
+            op=lambda col: col.decks.current(),
+            success=lambda deck: self._check_deck_and_draw(deck, links),
+        ).run_in_background()
```

Comment:
QueryOp is the best way to solve this, but there are two other existing DB queries in the same function (have_buried(), and another dyn check). Rather than add an extra query, I think we either need to a) assign the dyn check above to a variable and reuse it (pros: minimum code change before a release, cons: keeps accessing data on the main thread), or b) rewrite the whole routine so the QueryOp fetches all the needed data up-front.

Reply:
(thank you for the fix by the way :-)